src/features/broker-protection/utils.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * Get a single element.
+ *
+ * @param {Node} doc
+ * @param {string} selector
+ * @return {HTMLElement | null}
+ */
+export function getElement (doc = document, selector) {
+    if (isXpath(selector)) {
+        return safeQuerySelectorXPath(doc, selector)
```

Comment:
What does the 'safe' prefix represent?

Reply:
just that each of the calls is wrapped in a try-catch with a `null` return. Mostly because of throwing methods like querySelector.

Reply:
resolving the conversation for the sake of merging, but we can come back and re-name these if you think there'a a better alternative :)